assets/js/models.js

```diff
@@ -4,7 +4,7 @@ const api = document.getElementsByClassName("mr-component")[0].id;
 console.log(api)
 const typesenseInstantsearchAdapter = new TypesenseInstantSearchAdapter({
   server: {
-    apiKey: "EzBPJyIM1x2jv1C8oplyUjeONokTcJcZ", // Be sure to use an API key that only allows search operations
+    apiKey: "JhUowFH6ERd20FTIIzpTZtfDWFUp6lIs", // Be sure to use an API key that only allows search operations
```

Comment:
Just checking: This only allows search operations right? :)

Reply:
Yes